### ISSUE TITLE
Update troubles

### ISSUE THREAD

**Issue by musicpanda** [thirtybees/thirtybees#1662]
I  recently did an update of the modules and found two small issues:

 - The clickable module names that need an upgrade no longer work. In the past when you clicked the link that module was updated. Now nothing happens. 
![klikbaar](https://user-images.githubusercontent.com/13930107/228486358-7ca3c7a4-5d6f-4074-9f36-b9f9344bf1b1.jpg)

 - tbupdater gives an error when it is updated 
![update tbupdater](https://user-images.githubusercontent.com/13930107/228487114-e229125a-741e-4eae-912d-3d1d5454bc50.jpg)

**Comment by getdatakick** [comment]
The list of modules available for upgrade work the same as it always did. When you click on the link you will be redirected to module list with **&anchor=ModuleName** added in url. This will result in selecting module in the list only. Module will not be updated unless you click on **Update it!** button.

![image](https://user-images.githubusercontent.com/34651851/228575909-5780d201-03a0-4094-83a0-26cbc9e1f9bf.png)

I have created bug for the other issue in tbupdater module: thirtybees/tbupdater#19

**Comment by getdatakick** [comment]
Can't reproduce this on clean installation. Do you have any javascript errors in console?

This is very simple functionality that depends only on javascript. Template in back office only populates `anchor` javascript variable, and if exists, filtering takes place automatically.

**Comment by musicpanda** [comment]
> Can't reproduce this on clean installation. Do you have any javascript errors in console?
> 
> This is very simple functionality that depends only on javascript. Template in back office only populates `anchor` javascript variable, and if exists, filtering takes place automatically.

This shop is now updated. I will report it when the problem appears again.

**Comment by musicpanda** [comment]
Inspired by what you wrote I did some search for javascript errors. I found that indeed this shop has an error on the homepage: tools.js reports that the variable priceDisplayPrecision is not defined. I am still unsure what this could be.

I found that my localhost copy of the shop still had the problem of the module not being shown after being clicked. When I looked for javasript errors I found none. The url used is http://localhost/_testshop/admin123/index.php?controller=AdminModules&token=83bfbd02b0b7ad9b885c9c3475a23d80&anchor=Cronjobs

I have attached the html that it returns. Load it as a html file and you will see what I see.
[cronpage.txt](https://github.com/thirtybees/thirtybees/files/11103883/cronpage.txt)
